okio-files/src/commonMain/kotlin/okio/Filesystem.kt

```diff
@@ -109,11 +109,20 @@ abstract class Filesystem {
   @Throws(IOException::class)
   abstract fun delete(path: Path)
 
+  /**
+   * Returns a writable temporary directory on the current file system.
+   * This is the java.io.tmpdir system property on the JVM platform and the TMPDIR environment variable on the POSIX platform
+   */
+  internal abstract fun temporaryDirectory(): Path
```

Comment:
In a follow up I’ll move this elsewhere. Otherwise users can’t extend this class because they can’t implement this method.

Reply:
Makes sense 👍 . Thanks!